k4Reco/Overlay/components/OverlayTiming.h

```diff
@@ -140,6 +141,9 @@ struct OverlayTiming : public k4FWCore::MultiTransformer<retType(
       this, "TimeWindows", std::map<std::string, std::vector<float>>(), "Time windows for the different collections"};
   Gaudi::Property<bool> m_allowReusingBackgroundFiles{
       this, "AllowReusingBackgroundFiles", false, "If true the same background file can be used for the same event"};
+  Gaudi::Property<bool> m_copyCellIDMetadata{this, "CopyCellIDMetadata", false,
+                                             "If metadata is found in the signal file, copy it to the output file, "
```

Comment:
```suggestion
                                             "If metadata is found in the signal file, copy it to the output file, "
```

Which names are replaced?
And is there ever a case where we wouldn't want this to be done?

Reply:
All the ones that are `InputCollection__CellIDEncoding` (the convention in DD4hep) are changed to `OutputCollection__CellIDEncoding`. If for some reason the convention is different then I guess we wouldn't want this but there is no general way of finding the metadata associated to an input collection since from the algorithm there isn't a way of knowing all the names of the metadata parameters (which would allow for looking for a name that contains the current input collection). If the option is removed then there would be some warnings if the metadata paremeters are not found.